experiments/202005-pyodide-settrace/index.html

```diff
@@ -0,0 +1,40 @@
+<html>
+  <head>
+    <script src=" https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js"></script>
+  </head>
+  <body>
+      <script type="text/javascript">
+            const start = new Date().getTime()
+            languagePluginLoader.then(function () {
+                pyodide.runPython("print('hello world')");
+                console.log(new Date().getTime() - start)
+
+                pyodide.runPython(`
```

Comment:
i'll pull this up in intellij, but does it do a good job for editing embeded python?

Reply:
No this is just a sample of running some py code in the most basic way possible, I think we'd use a completely different means of getting py code from a browser-based editor to the py interpreter.

Very possibly by [direct filesystem manipulation](https://emscripten.org/docs/api_reference/Filesystem-API.html#id2).